docs/usage.rst

```diff
@@ -257,6 +257,14 @@ each scenario instead of flushing the entire database. Tests run much quicker,
 however HTTP server is not started and therefore web browser automation is
 not available.
 
+``--behave-runner``
```

Comment:
Didn't we agree to use the `--behave-` prefix _exclusively_ for arguments that are passed unchanged directly to `behave`? This makes things easier to understand. Good rules need no exceptions.

Reply:
Done.